packages/ruby/lib/readme/har/response_serializer.rb

```diff
@@ -48,7 +48,7 @@ def json_content
           size: @response.content_length,
           text: Har::Collection.new(@filter, parsed_body).to_h.to_json
         }
-      rescue
+      rescue StandardError
```

Comment:
As far as I can tell all errors extend `StandardError`. We don't have any custom error classes in the library so this seems fine?

Reply:
:shrug:

Reply:
I'm going to revert this because I'm paranoid that we won't catch some obscure error and break someones app.

Reply:
I believe the default for `rescue` is `StandardError`, it should be equivalent.